autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/AutofillScreens.kt

```diff
@@ -56,3 +61,24 @@ enum class AutofillSettingsLaunchSource {
     InternalDevSettings,
     Unknown,
 }
+
+@Parcelize
+/**
+ * The status of the privacy protections for the current page
+ */
+sealed interface PrivacyProtectionStatus : Serializable, Parcelable {
```

Comment:
do we need a sealed class for this? Seems you are just trying to know if protections are enabled or not. Will it be enough with a boolean?

Reply:
Removed